openlibrary/templates/type/author/view.html

```diff
@@ -217,7 +217,7 @@ <h3>$_('Links') <span class="gray small sansserif">($_('outside Open Library'))<
                 <ul class="booklinks sansserif">
                 $if page.remote_ids:
                     $if page.remote_ids.isni:
-                        $ isni = page.remote_ids.isni
+                       $ isni = page.remote_ids.isni
```

Comment:
What tool (preprocessor?) did this cause to fail? Seems like needlessly strict syntax.